### ISSUE TITLE
Remove users from database from during spam flood

### ISSUE THREAD

**Issue by StevenPuttemans** [opencv-infrastructure/answers.opencv.org#34]
The database still contains a horrific amount of 1 post sleeper cells, waiting to do a spam attack: http://answers.opencv.org/users/?sort=reputation&page=2791

Can we delete all 1 karma users from database one time, and get it cleaned up again? Just looped through users, it seems the list of unusable accounts starts around
http://answers.opencv.org/users/?sort=reputation&page=190 

And yes I am aware we will delete some accounts, but those will have to live up with it and just remake them. We could be removing around 64000 false accounts this way ...

**Comment by LaurentBerger** [comment]
I have found  gardezi http://answers.opencv.org/users/?sort=reputation&page=252 
2 badges and question http://answers.opencv.org/question/42937/integral-image/ 
Is it possible to delete user without question?

**Comment by StevenPuttemans** [comment]
Ow heck, we could even delete only after like lets say, page 1000. That would save more of those small cases.

**Comment by LaurentBerger** [comment]
I have just create an user testuser. I cannot find it in page list.

**Comment by StevenPuttemans** [comment]
> I have just create an user testuser. I cannot find it in page list.

Thats because you need an accepted post before you are added first!

**Comment by codestamp** [comment]
@StevenPuttemans  my account has been blocked. Cannot comment or ask questions. Can you please re-instate it.

**Comment by LaurentBerger** [comment]
Sorry. what's your user name on http://answers.opencv.org ?

**Comment by codestamp** [comment]
My username is **munesh**

**Comment by LaurentBerger** [comment]
@codestamp Do you remember your last question?


@mshabunin @berak

**Comment by codestamp** [comment]
No, but I want to comment and ask question on the following link
http://answers.opencv.org/question/89050/passing-an-array-of-cvgpumat-to-a-cuda-kernel/

**Comment by codestamp** [comment]
By the way now I remember, my question was on the blocking of my account on OpenCV itself.

**Comment by mshabunin** [comment]
I've unblocked user 'munesh', sorry for inconvenience.

**Comment by codestamp** [comment]
Many thanks @mshabunin . It's a big help.
